network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion_section_3.html

```diff
@@ -29,12 +29,11 @@ <h2 class="heading-2 mb-3">{% trans "We found that the rate of regrets is 60% hi
 
 <div class="row flex-column-reverse flex-lg-row">
   <div class="col-12 col-lg-5">
-    {% trans "36" as 36_percent %}
     {% trans "of non-english videos were pandemic-related" as of_non_english_videos %}
-    {% trans "14" as 14_percent %}
```

Comment:
**Issue** I don't think these lines should be removed. I now get 0 in both stat elements.

Reply:
Thanks, I forgot to replace the variables with actual values!